ivy/data_classes/array/experimental/layers.py

```diff
@@ -1117,3 +1117,45 @@ def rfftn(
             The result of the RFFT operation.
         """
         return ivy.rfftn(self._data, s=s, axes=axes, norm=norm, out=out)
+
+    def max_unpool1d(
+        self: ivy.Array,
+        indices: ivy.Array,
+        kernel_size: Union[Tuple[int], int],
+        strides: Union[int, Tuple[int]] = None,
```

Comment:
Just noticed you need to add `/, *,` before the start of the keyword default arguments.